soc/arm64/arm/fvp_aemv8a/soc.h

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Will this file provide anything useful in the future?  If not, I'd just put the needed #includes in mpu_regions.c above.

Reply:
Not currently. I'll delete it.

Reply:
Oh, well, it turns out that this file is included by `drivers/serial/uart_pl011.c` among others so better leave it in place.

Reply:
10-4.  I don't know if my approval means anything but I'll give it my +1.

Reply:
It's really appreciated nonetheless :)